ui/src/common/layouts/CollectionsMenu/index.js

```diff
@@ -0,0 +1,3 @@
+import CollectionsMenu from './CollectionsMenu';
+
+export default CollectionsMenu;
```

Comment:
I don't like this end of the line but is everywhere i guess
 :P

Reply:
well it's `js`, you have to export to be able import 😄

Reply:
<img width="53" alt="Screenshot 2020-03-18 at 14 38 40" src="https://user-images.githubusercontent.com/4287772/76966418-39f12480-6926-11ea-9ebd-fd09c0311875.png">

Reply:
I didn't mean the export :P